test/hdt-test.js

```diff
@@ -120,51 +120,47 @@ describe('hdt', function () {
           });
       });
       it('Should get suggestions for literals', function (done) {
-        document.getSuggestions('"a', { position : 'object' },
+        document.findTerms({ prefix: '"a', position : 'object' },
           function (error, suggestions) {
             if (error) return done(error);
             suggestions.should.have.lengthOf(8);
             done();
           });
       });
-      // Skipping, as this needs an hdt-cpp submodule that has this fix:
-      // https://github.com/rdfhdt/hdt-cpp/pull/87
-      it.skip('Should return suggestions on empty match', function (done) {
-        document.getSuggestions('', { position: 'object' },
+      it('Should 0 results on empty match', function (done) {
+        document.findTerms({ prefix: '', position: 'object' },
           function (error, suggestions) {
             if (error) return done(error);
-            suggestions.should.have.lengthOf(100);
+            suggestions.should.have.lengthOf(0);
             done();
           });
       });
-      // Skipping, as this needs an hdt-cpp submodule that has this fix:
-      // https://github.com/rdfhdt/hdt-cpp/pull/87
-      it.skip('Should return suggestions on null match', function (done) {
-        document.getSuggestions(null, { position: 'object' },
+      it('Should 0 results when prefix is not defined', function (done) {
+        document.findTerms({ position: 'object' },
           function (error, suggestions) {
             if (error) return done(error);
-            suggestions.should.have.lengthOf(100);
+            suggestions.should.have.lengthOf(0);
             done();
           });
       });
       it('Should return 0 results on negative limit', function (done) {
-        document.getSuggestions('http://example.org/', { limit:-1, position: 'object' },
+        document.findTerms({ prefix: 'http://example.org/', limit:-1, position: 'object' },
```

Comment:
Nitpick: missing space after colon here (linter should have caught this, so I need to check that).